Source/JavaScriptCore/dfg/DFGAdaptiveStructureWatchpoint.cpp

```diff
@@ -29,6 +29,7 @@
 #if ENABLE(DFG_JIT)
 
 #include "CodeBlock.h"
```

Comment:
Remove this line when Inlines.h header is included.

Reply:
Done here and elsewhere.